ui/js/controllers/perf/graphs.js

```diff
@@ -339,8 +339,11 @@ perf.controller('GraphsCtrl', [
                                     series.projectName, rev).then(
                                         function(resultSets) {
                                             var resultSetId = resultSets[0].id;
```

Comment:
Let's remove this temporary as part of this patch, that way you can use resultSetId insisde your function without a problem. E.g.:

```
function(resultSetId, index) {
   return resultSets[0].id == resultSetId ? index : null;
}
```